dfx.json

```diff
@@ -192,9 +192,10 @@
   },
   "networks": {
     "local": {
-      "bind": "0.0.0.0:8080",
-      "proxy": {
-        "domain": "0.0.0.0"
+      "bind": "127.0.0.1:8080",
+      "type": "ephemeral",
+      "replica": {
+        "subnet_type": "system"
```

Comment:
Do we need this?

Reply:
Sure I can remove it - just reverting to what we had before